Tasks/MavenV2/mavenutil.ts

```diff
@@ -237,6 +240,15 @@ async function collectFeedRepositories(pomContents:string): Promise<any> {
                                 ? repo.id[0]
                                 : repo.id
                             });
+                        continue;
+                        }
+                        if (url && url.toLowerCase().includes(oldCollectionName)) {
+                            tl.warning('Your pom file is still using the old style organization urls, please switch to the new url styles and update your pom file. Using credentials for old style url: ' + url);
```

Comment:
Can this warning (here and in the other files) be made localizable, and tweaked as follows?

`Your POM file uses the old style of Azure DevOps organization URLs (i.e. https://*.visualstudio.com). Update your POM file to use the new URL style (https://dev.azure.com/*). Using credentials for the old URL style: ' + url`

Reply:
@DavidStaheli Thanks, I updated the PR with the new localized string.